packages/experimental/AppearanceAdditions/src/NativeAppearanceAdditions.ts

```diff
@@ -0,0 +1,19 @@
+import { SizeClass, UserInterfaceLevel } from './NativeAppearanceAdditions.types';
+
+export const NativeAppearanceAdditions = {
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  addListener: (_: string) => {},
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  removeListeners: (_: number) => {},
+  horizontalSizeClass: () => {
+    console.warn('NativeAppearanceAdditions is only available on iOS');
+    return 'regular' as SizeClass;
+  },
+  userInterfaceLevel: () => {
+    console.warn('NativeAppearanceAdditions is only available on iOS');
+    return 'base' as UserInterfaceLevel;
+  },
```

Comment:
quick gut check we don't get these warnings on boot on macOS?

Reply:
The intent is that these functions would only be called from iOS components and theme-making code, but I'm spinning up a build to verify now.

Reply:
Confirmed! No warnings, only joy.